src/index.html

```diff
@@ -1,3 +1,4 @@
+<!-- #region head -->
 <!doctype html>
```

Comment:
The `<!doctype html>` declaration should be the very first line in your HTML document, before the `<html>` tag.